utility/src/main/java/org/oppia/util/parser/SvgDrawableTranscoder.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.util.parser
+
+import android.graphics.drawable.PictureDrawable
+import com.bumptech.glide.load.Options
+import com.bumptech.glide.load.engine.Resource
+import com.bumptech.glide.load.resource.SimpleResource
+import com.bumptech.glide.load.resource.transcode.ResourceTranscoder
+import com.caverock.androidsvg.SVG
+
+/** SvgDrawableTranscoder converts SVG to PictureDrawable. */
+class SvgDrawableTranscoder : ResourceTranscoder<SVG?, PictureDrawable?> {
+  override fun transcode(
+    toTranscode: Resource<SVG?>,
+    options: Options
+  ): Resource<PictureDrawable?>? {
+    val svg: SVG = toTranscode.get()
+    val picture = svg.renderToPicture()
```

Comment:
By pre-rendering it to a Picture does the SVG still scale without artifacts when rendered in a view?

Reply:
Scaling looks  fine. I tried loading high and low resolution images. It renders smoothly without blurring image.

Reply:
Hmm I guess because a picture object stores draw calls that this scales as expected. In that case, maybe it's best for us to pass a Picture along the pipeline since it probably will be a bit more performant to only render the picture once per load of the image rather than each time we actually want to render it.